src/main/groom/mutations.cljc

```diff
@@ -3,10 +3,10 @@
             [clojure.spec.alpha :as s]
             [clojure.string :as str]
             [groom.queries :as q]
-            [groom.specs.v1]))
+            [groom.specs.v1 :as v1]))
 
 (s/fdef graphql-bound-mutation-params
-  :args (s/cat :params (s/spec ::groom.specs.v1/mutation-call-params))
```

Comment:
I think the queries and mutations namespaces are perfect candidates for cljc. When converting them to cljc I noticed that the double colon in front of `::groom.specs.v1/mutation-call-params` fails in clojure, but not in clojurescript. I choose to use the alias instead of the fully qualified keywords, because I think the we will use the v1 namespace quite often here and in the queries namespace.